File: src/main/java/org/smarthata/rest/PingController.java
package org.smarthata.rest;

import org.springframework.beans.factory.annotation.Value;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class PingController {

    @Value("${application.name}")
    private String applicationName;

    @Value("${build.version}")
    private String buildVersion;

    @GetMapping
    public String version() {
        return applicationName + ":" + buildVersion;
    }
}

File: src/main/kotlin/org/smarthata/rest/PingController.kt
package org.smarthata.rest

import org.springframework.beans.factory.annotation.Value
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class PingController(
    @Value("\${application.name}") private val applicationName: String,
    @Value("\${build.version}") private val buildVersion: String,
) {
    @GetMapping("/")
    fun version(): String = "$applicationName:$buildVersion"
}